io/net/net_data.py

```diff
@@ -76,13 +84,11 @@ def testbigping(self):
         check with different maximum transfer unit values
         '''
         tmp = "grep -w -B 1 %s" % self.peer
-        cmd = "\`ifconfig | %s | head -1 | cut -f1 -d' '\`" % tmp
+        cmd = "`ifconfig | %s | head -1 | cut -f1 -d' '`" % tmp
```

Comment:
same variable can be used instead of new one for using it once.
`cmd = "grep -w -B 1 %s" % self.peer`
`cmd = "ifconfig | %s | head -1 | cut -f1 -d'  '" % cmd`